sdk/cosmos/azure-cosmos/setup.py

```diff
@@ -66,6 +66,8 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
Good catch! We can actually add 3.12 as well....
```suggestion
        "Programming Language :: Python :: 3.11",
        "Programming Language :: Python :: 3.12",
```